db/migrate/20190815191552_add_message_to_subscription_contents.rb

```diff
@@ -0,0 +1,9 @@
+class AddMessageToSubscriptionContents < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :subscription_contents,
+                  :message,
+                  type: :uuid,
+                  foreign_key: { on_delete: :restrict },
+                  index: false
```

Comment:
Just an FYI that when I wrote this before I wasn't quite sure whether adding a foreign key without an index was a table blocking operation. So just a heads up to be cautious with the deploy.

Reply:
It looks like it can cause problems, so I'm going to split this up into two migrations: https://github.com/ankane/strong_migrations#good-5

Reply:
I've pushed up a change to split this up into two migrations if you'd be able to take another look.